src/main/cpp/geo/Profile.h

```diff
@@ -0,0 +1,31 @@
+#ifndef GEO_PROFILE_H_INCLUDED
+#define GEO_PROFILE_H_INCLUDED
+
+#include "City.h"
+#include <memory>
+#include <fstream>
+
+namespace stride {
+namespace geo {
+
+class Profile;
+using ProfileRef = std::shared_ptr<const Profile>;
+
+class Profile
+{
+public:
+	Profile(std::vector<City> cities) : cities(cities) {}
```

Comment:
Can you change `std::vector<City>` to `const std::vector<City>&`? Right now, the `cities` vector is copied *twice:* once when it is passed as an argument to `Profile::Profile` and again when field `cities` is initialized with parameter `cities`.